.github/workflows/deploy-to-beta.yml

```diff
@@ -0,0 +1,103 @@
+name: Deploy to beta on trigger
```

Comment:
Maybe `./github/workflows/deploy/beta.yml` would be better?

Reply:
Moved under `deploy` dir. Renamed also.